### PR TITLE
Update to new Storefront model

### PR DESCRIPTION
**Pull request by tamorim**
- Removed the `assign` stuff
- Add routes, areas
- Fix component json

### CODE REVIEW

**Comment by brenoc** [comment]
Everything looks good! Just one thing: [README.md](https://github.com/vtex-apps/product-page-wrapper/blob/fix/new-storefront-model/README.md)

**Comment by tamorim** [comment]
@brenoc Updated!

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This app provides a helpful wrapper for the product page.
 
 Add "vtex.product-page-wrapper" as your app dependency.
 
-In your client React component call the action `actions.ProductPageWrapperActions.assign()` passing your React component as a param.
+To ensure that your product page will be used by the `ProductPageWrapper`, declare your page at `storefront/product/page.json` on your project.
```
Comment (brenoc) [inline README.md]: Ops, it's: `storefront/areas/product/page.json`

**Comment by brenoc** [comment]
Oh, and one more thing, remove this line: https://github.com/vtex-apps/product-page-wrapper/blob/master/examples/ProductPage.js#L16

**Comment by tamorim** [comment]
@brenoc Updated!

**Pull request merged**